include/cli/modules/deploy.php

```diff
@@ -95,21 +95,19 @@ function clean($local, $destination, $root, $recurse=0, $exclude=false) {
         }
     }
 
-    function writeManifest($root) {
+    private function writeManifest() {
```

Comment:
> Why the change to `private` here
It is not used in other classes, the time you need this as extend lets see that case.

I prefer to make everything private if its not used, by doing that I know for sure its not used somewhere else (makes is less scary to edit the code!). Also, it forces you to think, maybe extend the class is not the answer but extract it from the class to reuse it.

So, if you wanna be "flexible" make sure you give your functions the right task and only that task. This way you can reuse it easily and you are flexible again! 😄 

But yea, that's my opinion and you guys are the repo leaders, so I go with what you want, but please, consider it.